lib/webpack-cli.js

```diff
@@ -0,0 +1,77 @@
+const { join } = require("path");
+const ErrorHelper = require("./utils/error-helper");
+
+class webpackCLI extends ErrorHelper {
+	constructor() {
+		super();
+		this.groupMap = new Map();
+		this.groups = [];
+		this.processingErrors = [];
+		this.shouldUseMem = false;
+	}
+	setMappedGroups(args, yargsOptions) {
+		const { _all } = args;
+		Object.keys(_all).forEach(key => {
+			this.setGroupMap(key, _all[key], yargsOptions);
+		});
+	}
+	setGroupMap(key, val, yargsOptions) {
+		const opt = yargsOptions.find(opt => opt.name === key);
+		const groupName = opt.group;
+		const namePrefix = groupName.slice(0, groupName.length - 9).toLowerCase();
+		if (this.groupMap.has(namePrefix)) {
+			const pushToMap = this.groupMap.get(namePrefix);
+			pushToMap.push({ [opt.name]: val });
+		} else {
+			this.groupMap.set(namePrefix, [{ [opt.name]: val }]);
+		}
+	}
+	formatDashedArgs() {}
+
+	resolveGroups() {
+		for (const [key, value] of this.groupMap.entries()) {
+			const fileName = join(__dirname, "groups", key);
+			const GroupClass = require(fileName);
+			const GroupInstance = new GroupClass(value);
+			this.groups.push(GroupInstance);
+		}
+	}
+
+	runOptionGroups() {
+		return this.groups
+			.map(Group => {
+				const res = Group.run();
+				if (res.err) {
+					this.processingErrors.push(res.err);
+					delete res.err;
+				}
+				return res;
+			})
+			.filter(e => e);
+	}
+
+	normalizeGroup(group) {
+		return this.arrayToObject(group);
+	}
+
+	async run(args, yargsOptions) {
```

Comment:
I get errors from `eslint` saying that this is not supported, because we support, as min version, `6.11.5`, which doesn't have `async`/`await`

Reply:
Hm that is bad, will look into it